core/app/models/spree/order_updater.rb

```diff
@@ -48,17 +47,6 @@ def update_totals
       update_adjustment_total
     end
 
-
-    # give each of the shipments a chance to update themselves
-    def update_shipments
```

Comment:
here is the issue that led to this code being where it's al today. https://github.com/spree/spree/pull/5235